Sources/GoodReactor/GoodReactor.swift

```diff
@@ -159,6 +164,10 @@ public extension GoodReactor {
         set { MapTables.cancellables.setValue(newValue, forKey: self) }
     }
 
+    func start() {
```

Comment:
Lets add documentation to the `start()` method or above the whole class with more description, because this function is a core to make Reactor work without `bindState()` function that we used in UIKit implementation.

> The `start()` method creates an initial publisher and automatically subscribes to that publisher. 
> Without subscription the State Stream would not be created, therefore - reactor actions would not be called (transform, mutate etc.).

cc @smajdalf